docs/src/token-2022/status.md

```diff
@@ -12,9 +12,9 @@ Here is the general program timeline and rough ETAs:
 
 | Issue | ETA |
 | --- | --- |
-| Finish program | end of May |
-| Final audit | end of June |
-| Mainnet recommendation | middle of July |
+| Finish program | end of June |
+| Final audit | end of July |
+| Mainnet recommendation | middle of August |
```

Comment:
I'd push this out further as it current depends on adoption of v1.16 and feature gate activations.

Actually explicitly calling the the v1.16 dep and the corresponding feature gates here would be super, that gives people more vis into the real release constraints and a handy excuse for why the estimate date might be pushed out later :)

Reply:
Ah I see. My initial thinking was that we can officially recommend it on mainnet before confidential transfers are available, but it seems like that'll be too confusing.

Thanks for the fly-by review!